test/easyconfigs/easyconfigs.py

```diff
@@ -474,8 +474,8 @@ def check_dep_vars(self, gen, dep, dep_vars):
                             'NGSpeciesID-0.1.1.1-']),
                 # medaka 1.4.3 (foss/2019b) depends on TensorFlow 2.2.2
                 ('2.2.2;', ['medaka-1.4.3-']),
-                # medaka 1.4.3 (foss/2020b) depends on TensorFlow 2.2.3
-                ('2.2.3;', ['medaka-1.4.3-']),
+                # medaka 1.4.3 (foss/2020b) and thus artic-ncov2019-2021.06.24 depend on TensorFlow 2.2.3
+                ('2.2.3;', ['medaka-1.4.3-', 'artic-ncov2019-2021.06.24artic-ncov2019-2021.06.24']),
```

Comment:
```suggestion
                ('2.2.3;', ['medaka-1.4.3-', 'artic-ncov2019-2021.06.24-']),
```

Reply:
I was unsure whether or not there needs to be a '-' so I picked the wrong option.